src/lib/components/submit/Submit.svelte

```diff
@@ -42,7 +47,7 @@
 			return;
 		}
 
-		step = 'write';
+		step = 'select';
```

Comment:
This is actually not correct, the step select should only appears in the wizard and as first step if the governance canister is NNS. For any Snses this step should not exist.

We can add a TODO and tackle this as next task once this PR is merged to not add extra complexity here.